pkg/reconciler/v1alpha1/route/route.go

```diff
@@ -222,6 +227,13 @@ func (c *Reconciler) reconcile(ctx context.Context, r *v1alpha1.Route) error {
 
 	r.Status.InitializeConditions()
 
+	if r.GetDeletionTimestamp() != nil {
+		// Check for a DeletionTimestamp.  If present, elide the normal reconcile logic.
+		// Instead, if our Finalizer is first, delete the ClusterIngress for this Route
```

Comment:
Line #232-#233 may better be moved to the reconcileDeletion() func?

Reply:
Done.